packages/web/src/experience/eventkit/AmbassadorPage.tsx

```diff
@@ -0,0 +1,14 @@
+import * as React from 'react'
+import Page, { CIRCLES_PATH, ROUTE_TO_TITLE } from 'src/experience/eventkit/Page'
+import Section from 'src/experience/eventkit/Section'
+import { hashNav } from 'src/shared/menu-items'
+
+const sections = [hashNav.eventCircles.overview].map((id) => {
```

Comment:
Why the [].map pattern here for a single item?

Reply:
was just less work to copy the pattern. and less work in the future when they want to add another section, (which i believe they plan on doing)